<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BreezeExceptionDialog</class>
 <widget class="QDialog" name="BreezeExceptionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>397</width>
    <height>346</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Window Identification</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>&amp;Matching window property: </string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>exceptionType</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Regular expression &amp;to match: </string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>exceptionEditor</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="detectDialogButton">
        <property name="text">
         <string>Detect Window Properties</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="QLineEdit" name="exceptionEditor">
        <property name="showClearButton" stdset="0">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="QComboBox" name="exceptionType">
        <item>
         <property name="text">
          <string>Window Class Name (X11 only)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Window Title</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Decoration Options</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="1">
       <widget class="QComboBox" name="borderSizeComboBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <item>
         <property name="text">
          <string comment="@item:inlistbox Border size:">No Border</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string comment="@item:inlistbox Border size:">No Side Borders</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string comment="@item:inlistbox Border size:">Tiny</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string comment="@item:inlistbox Border size:">Normal</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string comment="@item:inlistbox Border size:">Large</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string comment="@item:inlistbox Border size:">Very Large</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string comment="@item:inlistbox Border size:">Huge</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string comment="@item:inlistbox Border size:">Very Huge</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string comment="@item:inlistbox Border size:">Oversized</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="4" column="0" colspan="2">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QCheckBox" name="hideTitleBar">
        <property name="text">
         <string>Hide window titlebar</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QCheckBox" name="borderSizeCheckBox">
        <property name="text">
         <string>Border size:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QCheckBox" name="opaqueTitleBar">
        <property name="text">
         <string>Opaque titlebar/header</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>BreezeExceptionDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>252</x>
     <y>342</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>BreezeExceptionDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>320</x>
     <y>342</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>borderSizeCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>borderSizeComboBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>125</x>
     <y>162</y>
    </hint>
    <hint type="destinationlabel">
     <x>316</x>
     <y>163</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
